acme/acme/client.py

```diff
@@ -0,0 +1,582 @@
+"""ACME client API."""
+import datetime
+import heapq
+import logging
+import time
+
+from six.moves import http_client  # pylint: disable=import-error
+
+import OpenSSL
+import requests
+import sys
+import werkzeug
+
+from acme import errors
+from acme import jose
+from acme import jws
+from acme import messages
+
+
+logger = logging.getLogger(__name__)
+
+# Python does not validate certificates by default before version 2.7.9
+if sys.hexversion < 0x20709F0 :
```

Comment:
Nice find! Could you please send this update in a separate PR to `master`? Can we use `sys.version_info < (2, 7, 9)` instead?